core/domain/state_domain.py

```diff
@@ -786,16 +786,16 @@ def to_dict(self):
         """Returns a dict representing this WrittenTranslations domain object.
 
         Returns:
-            dict. A dict, mapping content_id to a dict containing language_code
-            as a key and written translation dict as a value.
+            dict. A dict, mapping all fields of WrittenTranslations instance.
         """
-        written_translations_dict = {}
+        written_translations_dict = {'translations_mapping': {}}
+        translations_mapping = written_translations_dict['translations_mapping']
```

Comment:
It would be cleaner to start by defining this as {}, populating it, and then only assembling the written_translations_dict at the end (i.e. delete the line above and do it at the end of the method). That way, the reader doesn't have to keep remembering which variables are passed by reference.